gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -436,12 +435,16 @@ def add_annotation_data(self, proc_var, annotation_field_name, data):
             self._add_ambiguous_fields(annotations_list, ambiguous)
           alt._info[annotation_field_name] = annotations_list
 
-  def _find_common_alt_ref_prefix(self, proc_var):
+  def _find_common_alt_ref_prefix_char(self, proc_var):
     # type: (ProcessedVariant) -> str
-    alt_list = [
-        alt.alternate_bases for alt in proc_var._alternate_datas]
-    alt_list.append(proc_var.reference_bases or '')
-    return os.path.commonprefix(alt_list)
+    if proc_var.reference_bases:
```

Comment:
I think it's more readable to do `if not proc_var.reference_bases: return ''` and then have the rest of the logic (it's strange how the first if doesn't return and then switches the control to the for loop below it). It also looks like you are either returning empty string or the first character of reference_bases, so, it may be better to special case all cases where you return empty string first, and then `return proc_var.reference_bases[0]` (this latter part is just a suggestion though)

Reply:
Done (the first part).